src/main/minio.js

```diff
@@ -1488,7 +1490,7 @@ export class Client {
       throw new TypeError('expires should be of type "number"')
     }
     var method = 'PUT'
-    var requestDate = Moment().utc()
+    var requestDate = new Date()
```

Comment:
This should be a non-issue as anytime we create a `Date` object, we only run `.toISOString()` on it. That function [will always return in UTC](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString), so we don't have to worry about this.